bin/sbank_pipe

```diff
@@ -37,13 +37,13 @@ from glue import pipeline
 from lalapps import inspiral
 
 def which(prog):
-    which = subprocess.Popen(['/usr/bin/which', prog], stdout=subprocess.PIPE)
-    out = which.stdout.read().strip()
-    if not out:
+    out = shutil.which(prog)
+    if out is None:
         print("ERROR: could not find %s in your path, have you built the proper software and source the proper env. scripts?" % (prog,prog), file=sys.stderr)
         raise ValueError
```

Comment:
Nitpick (last comment, I promise): remove the print and move the error message to the actual exception.

Reply:
The current code actually contains an error, by the way…

Reply:
Do you mean the old code has an error, or there's some problem here still?

Reply:
The original code did (one `%s` but two arguments). You change fixed it.